### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by aslanenver** [florisboard/florisboard#2126] [General]
When will the update arrive

**Comment by Mokura** [comment]
When the developer releases it, or when you fork the repo and submit an approved pull request.

**Comment by Silther** [comment]
It's still nice to know whether the app is still being developed or not.

**Comment by BHydden** [comment]
It is, they're just on a tricky section. More details are in the official matrix chat channel.

**Reply by Silther** [reply]
Is the matrix chat public?

**Reply by BHydden** [reply]
Yep just download any matrix client such as element or whatever. They're easily searchable by name.

**Comment by aslanenver** [comment]
Will there be an update?

**Reply by Mokura** [reply]
Every time someone asks this question, a month gets added to the roadmap.  /sarcasm

**Reply by Turhvjbufv** [reply]
#2197
